releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,7 +1,16 @@
-## Microsoft.Extensions.Configuration.AzureAppConfiguration
-### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
+# Microsoft.Extensions.Configuration.AzureAppConfiguration
+[Source code][source_code] | [Package (NuGet)][package]
 
-### 7.0.0 - November 21, 2023
+## 7.1.0-preview - January 19, 2024
+### Enhancements
+* The provider will now automatically discover other existing replicas and attempt to connect to them when the AppConfig store and any replicas passed by the user have failed. This is enabled by default and can be disabled by setting the new `AzureAppConfigurationOptions.ReplicaDiscoveryEnabled` property equal to `false`. [#442](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/442)
+* Added support for the `variants`, `allocation`, and `telemetry` properties for feature flags introduced in version 4.0.0-preview of the [Microsoft.FeatureManagement](https://github.com/microsoft/FeatureManagement-Dotnet/releases/tag/4.0.0-preview) library. [#476](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/476)
+
+### Bug Fixes
+* Fixed a bug where passing a null value for the `key` parameter to `AzureAppConfigurationRefreshOptions.Register` would throw a `NullReferenceException`. [#503](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/503)
+* Fixed a bug with `AzureAppConfigurationOptions.Select`, `FeatureFlagOptions.Select`, and `AzureAppConfigurationOptions.SelectSnapshot` where additional calls with the same filters after the first call were ignored. The provider will now correctly use only the last call with duplicate filters to preserve label precedence. [#490](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/490)
```

Comment:
> ... preserve label precedence.

Is it just label, not key and label?

Reply:
I was trying to describe the scenario where multiple select calls have the same key but different label, and the label used is prioritized based on the order. I could say "key and label" if that makes more sense?

Reply:
... or we simply say "... to preserve the expected precedence"?